package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "all-node-versions",
-  "version": "10.1.1",
+  "version": "11.0.0",
```

Comment:
There is a separate release command (documented [here](https://github.com/ehmicky/dev-tasks#gulp-releasepatch-gulp-releaseminor-gulp-releasemajor)) that I will perform after this PR is merged, which will bump the `package.json` `version` (among other things), so this PR does not need to do so.